sound/soc/sof/ipc4-topology.c

```diff
@@ -2871,7 +2871,7 @@ static int sof_ipc4_set_copier_sink_format(struct snd_sof_dev *sdev,
 	struct sof_ipc4_msg msg = {{ 0 }};
 
 	dev_dbg(sdev->dev, "%s set copier sink %d format\n",
-		src_widget->widget->name, sink_id);
+		src_widget->widget->name, sroute->src_queue_id);
```

Comment:
This is very confusing. Can you remind us in the commit message what the convention is for sinks/sources, having a log message that says 'sink N' where N is a source_queue_id is not really helping....

The gstreamer convention is that a sink receives data, but I think IPC4 provides data on a sink pin?

Reply:
The connection in IPC4 is:
src_widget.pin[sroute->src_queue_id] -> sink_widget.pin[sroute->dst_queue_id]
Before this patch we used the same queue_id (or pin_id) to get the format from the sink_widget, which was incorrect.

Reply:
@ujfalusi the problem I have is that the dev_dbg log mixes sink and source.

Reply:
it prints things correctly.
We are setting the src_widget's sink (pin) format, that is src_widget.pin[sroute->src_queue_id]
We do that by copying the input format of the destination widget, and we say that we cannot get it if it is not found.

But right, this is amazingly confusing..
the source widget have sink pin, which is connected to a sink widget's source pin, we refer to both as s_swidget / s_pin/s_queue all around to make things more fun :D

Reply:
@plbossart, I think the right course of action is to remove this print as such. We will print the connection anyways and this brings no value, we will print on error.